shared/utils.js

```diff
@@ -122,7 +122,8 @@ function getToolVersion(cmd) {
     var toolVersion;
     try {
 	 var result = runProcessThrowError(cmd, null, true /* return output */);
-        toolVersion = result.replace(/\r?\n|\r/, '').replace(/[^0-9\.]*/, '');
+        // Remove @salesforce/cli/ from the beginning of sf cli version.
+        toolVersion = result.replace(/\r?\n|\r/, '').replace(/[^0-9\.]*/, '').replace('@salesforce/cli/', '');
```

Comment:
This and the sfdx minVersion change from `6.0.0` to `2.0.0` are the important fixes for the migration.